Xamarin.Forms.Platform.iOS/CollectionView/ItemsViewLayout.cs

```diff
@@ -170,6 +170,10 @@ public void PrepareCellForLayout(ItemsViewCell cell)
 
 		public override bool ShouldInvalidateLayout(UICollectionViewLayoutAttributes preferredAttributes, UICollectionViewLayoutAttributes originalAttributes)
 		{
+			// This is currently causing an infinite layout loop on iOS 15 https://github.com/xamarin/Xamarin.Forms/issues/15230
+			if (preferredAttributes.RepresentedElementKind == "UICollectionElementKindSectionHeader" && Forms.IsiOS15OrNewer)
```

Comment:
this only affects iOS 15.4 or higher. Can we add this check so iOS 15.3.1 or lower versions are not affected?